### PR TITLE
optical_path_length_from_geometric_path_length_and_refractive_index

### PR DESCRIPTION
**Pull request by Darya2909**

### CODE REVIEW

File: symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless)
+
+# Description
+## The optical path length L between points A and B of a transparent medium is the distance over which light (optical radiation) has spread it would be in a vacuum for the same time it takes from A to B in the medium.
```
Comment (blackyblack) [inline symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py:6]: You still write about paths between points A and B, while I'm convinced it is the same path. Check and fix your description.

File: symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless)
+
+# Description
+## Optical path length is the length that light needs to travel through a vacuum to create the same phase difference as it would have when traveling through a given medium.
```
Comment (blackyblack) [inline symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py:6]: Could you elaborate on phase difference?

Comment (Darya2909) [reply symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py:6]: About the phase difference information is only about interference

File: symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py [hunk 3]
```diff
@@ -4,7 +4,10 @@
 
 # Description
 ## Optical path length is the length that light needs to travel through a vacuum to create the same phase difference as it would have when traveling through a given medium. 
-
+## If two light beams have common starting and ending points, then the difference in the optical path lengths of such rays is called the optical path difference.
+## The the optical path difference corresponds to the phase shift undergone by the light emitted from two previously coherent sources when passed through mediums of different refractive indices.
+## If the difference in the course of the waves is equal to an integer number of waves (i.e. an even number of half-waves), then an interference maximum is formed at the point of superposition of these waves. 
+## If the difference in the course of the waves is equal to an odd number of half-waves, then an interference minimum is formed at the point of superposition of these waves.
```
Comment (blackyblack) [inline symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py:10]: It is description of coherent waves interference. Unrelated to this law.
You should explain what the optical path is, or what phase difference is.

**Review by blackyblack — CHANGES_REQUESTED** [review]

File: symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py [hunk 4]
```diff
@@ -3,11 +3,7 @@
                            validate_output, dimensionless)
 
 # Description
-## Optical path length is the length that light needs to travel through a vacuum to create the same phase difference as it would have when traveling through a given medium. 
-## If two light beams have common starting and ending points, then the difference in the optical path lengths of such rays is called the optical path difference.
-## The the optical path difference corresponds to the phase shift undergone by the light emitted from two previously coherent sources when passed through mediums of different refractive indices.
-## If the difference in the course of the waves is equal to an integer number of waves (i.e. an even number of half-waves), then an interference maximum is formed at the point of superposition of these waves. 
-## If the difference in the course of the waves is equal to an odd number of half-waves, then an interference minimum is formed at the point of superposition of these waves.
+## The optical path length of light in a substance is numerically equal to the path length in a vacuum, which contains the same number of light waves as the geometric length in a substance.
```
Comment (blackyblack) [inline symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py:6]: Could you show me a source of this definition?

Comment (Darya2909) [reply symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py:6]: http://ostashevvb.spb.ru/Download/[07]%20Оптика.pdf page 17

**Review by blackyblack — APPROVED** [review]

**Pull request merged**